projects/plugins/jetpack/modules/memberships/class-jetpack-memberships.php

```diff
@@ -428,10 +440,19 @@ public static function get_blog_id() {
 	/**
 	 * Get the id of the connected payment acount (Stripe etc).
 	 *
-	 * @return int|void
+	 * @return bool
 	 */
-	public static function get_connected_account_id() {
-		return get_option( self::$connected_account_id_option_name );
+	public static function has_connected_account() {
+
+		// This is the primary solution.
+		$has_option = get_option( self::$has_connected_account_option_name, false ) ? true : false;
```

Comment:
Instead of using `false` for the fallback, could you use null and then return as long as `! is_null( $has_option )`?